content/en/service_management/case_management/_index.md

```diff
@@ -23,6 +23,8 @@ Case Management is not available in the {{< region-param key=dd_datacenter code=
 </div>
 {{% /site-region %}}
 
+{{< img src="/service_management/case_management/case_management_overview.png" alt="Overview of the Case Management page" style="width:100%;" >}}
```

Comment:
I recommend having the image after the ## Overview section (but that's my personal preference).